doc/PrairieDraw.md

```diff
@@ -1,6 +1,175 @@
 
 # PrairieDraw
 
+The PrairieDraw library (`PrairieDraw.js`) is a figure-drawing library which allows dynamic figures to be rendered through the `pl_prairiedraw_figure` custom element.
+
+## PrairieDraw figure element configuration
+
+The `pl_prairiedraw_figure` element is used to place figures within `question.html`. To make this element available for your course, a copy of the `pl_prairiedraw_figure` directory must be placed in the `elements` directory for your course.
+
+## `pl_prairiedraw_figure` element
+
+```html
+<pl_prairiedraw_figure script_name="drawFigure.js" params_names="r1,r2,isHorizontal" width="900" height="400" />
```

Comment:
In the element naming and attribute specification, the PL group has switched from snake-case in favor of kebab-case. (PR's https://github.com/PrairieLearn/PrairieLearn/pull/1171 and https://github.com/PrairieLearn/PrairieLearn/pull/1193) 

e.g.

`pl_prairiedraw_figure` => `pl-prairiedraw-figure`

`script_name` => `script-name`